plat/rockchip/common/plat_pm.c

```diff
@@ -412,6 +412,7 @@ const plat_psci_ops_t plat_rockchip_psci_pm_ops = {
 	.pwr_domain_suspend = rockchip_pwr_domain_suspend,
 	.pwr_domain_on_finish = rockchip_pwr_domain_on_finish,
 	.pwr_domain_suspend_finish = rockchip_pwr_domain_suspend_finish,
+	.pwr_domain_pwr_down_wfi = rockchip_pd_pwr_down_wfi,
```

Comment:
I am sorry that I reply it late. I do not find this error and rockchip_pd_pwr_down_wfi() is implement in the same file with (.pwr_domain_pwr_down_wfi = rockchip_pd_pwr_down_wfi); 
This patch needs to base on https://github.com/ARM-software/arm-trusted-firmware/pull/854.